### ISSUE TITLE
Converting dump into SQL results in missing semicolon and newline after `FROM stdin`

### ISSUE THREAD

**Issue by wnatrifork** [GreenmaskIO/greenmask#179]
Converting a dump into SQL file:

```
pg_restore 1724504511561 --file 1724504511561.sql
```

Note that semicolon and newline is missing after `FROM stdin`

```
--
-- Data for Name: "backlog_entry"; Type: TABLE DATA; Schema: "public"; Owner: "wna"
--

COPY "public"."backlog_entry" ("unique_id", "created", "year", "month", "department_id", "source") FROM stdin33cd6668-6dd7-4059-95a1-e5ae03027c04	2020-05-25 08:22:27.177641	2020	4	205e604f-14e4-4073-ab7f-886515ec4eea	recurring
```

Dumping database with pg_dump does not have this issue:

```
% pg_dump bireporting --format=d --file=bireporting
% pg_restore bireporting --file bireporting.sql
```

```
--
-- Data for Name: backlog_entry; Type: TABLE DATA; Schema: public; Owner: wna
--

COPY public.backlog_entry (unique_id, created, year, month, department_id, source) FROM stdin;
33cd6668-6dd7-4059-95a1-e5ae03027c04	2020-05-25 08:22:27.177641	2020	4	205e604f-14e4-4073-ab7f-886515ec4eea	recurring
```

**Comment by wwoytenko** [comment]
Thank you for reporting this. I've double-checked, and it seems you're correct. I will fix it.

**Comment by wwoytenko** [comment]
FIxed in #180
